ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/search-form-interactions/dropdown.search-form-interactions.less

```diff
@@ -0,0 +1,25 @@
+@{customElementNamespace}dropdown.is-searchFormInteractions {
```

Comment:
nit: `dropdown.is-search-form-interactions` for consistency

Reply:
Less file was not needed because search-form.less is dictating look and feel.  Files was removed